crates/primitives-traits/src/constants/mod.rs

```diff
@@ -10,6 +10,9 @@ pub const RETH_CLIENT_VERSION: &str = concat!("reth/v", env!("CARGO_PKG_VERSION"
 /// Minimum gas limit allowed for transactions.
 pub const MINIMUM_GAS_LIMIT: u64 = 5000;
 
+/// Maximum gas limit allowed for block.
+pub const MAXIMUM_GAS_LIMIT_BLOCK: u64 = 0x7fffffffffffffff;
```

Comment:
please also add this as number and that this is 2^63-1

Reply:
I added 2u64.pow(63) - 1 and added a comment to say its `0x7fffffffffffffff` -- let me know if thats what you meant ot it needs changing